### ISSUE TITLE
Suggestion: Transparent Background

### ISSUE THREAD

**Issue by naqaden** [shedaniel/ClothConfig#40]
If reasonable, it would be handy (depending on the mod) to not render the background at all. In my current case it would help players see changes to nameplates without backing out of the menu to evaluate, then navigating back in to tweak further.

This idea came from a glitch while playing with Cloth Config's demo configs. All the UI and shading rendered properly but the background was missing, so I could see the paused world behind. I didn't know it was an glitch at the time and haven't been able to reproduce it since.

Edit: I forgot to add, I've tried using 8-bit and 24-bit transparent textures as the background to achieve this effect but they result in opaque backgrounds. I only tried in 1.15.2.

**Comment by shedaniel** [comment]
It was not a glitch, it is an intended feature, https://github.com/shedaniel/ClothConfig/blob/eb887403227cb0075e3e37f56fc3927a4c98cdfe/src/main/java/me/shedaniel/clothconfig2/api/ConfigBuilder.java#L86

**Comment by naqaden** [comment]
Oh! When I checked CC's source to see how it was done, I found the [oak plank texture](https://github.com/shedaniel/ClothConfig/blob/b8e8f6c4db4fa7b58bc29bd4b54b5de774a83616/src/main/java/me/shedaniel/clothconfig2/ClothConfigInitializer.java#L176) set as the background and that plank texture is all I've seen since when trying to reproduce the transparent background.

Now to see if Auto Config Update can implement this transparency feature.
